src/build/arg/mod.rs

```diff
@@ -67,6 +67,7 @@ pub struct Arg<'help> {
     pub(crate) short: Option<char>,
     pub(crate) long: Option<&'help str>,
     pub(crate) aliases: Option<Vec<(&'help str, bool)>>, // (name, visible)
```

Comment:
@pksunkara Should we change this to `long_aliases`?

Reply:
@pickfire This was discussed in https://github.com/clap-rs/clap/issues/1896#issuecomment-623593526